modules/ROOT/pages/getting-started-libvirt.adoc

```diff
@@ -17,11 +17,13 @@ IGNITION_CONFIG="/path/to/example.ign"
 IMAGE="/path/to/image.qcow2"
 VM_NAME="fcos-test-01"
 RAM_MB="2048"
-DISK_GB="10"
+DISK_GB=10
```

Comment:
I don't understand, what broke with this?

Reply:
I was unable to reproduce my error when using variables. So this commit has to be deleted